src/objects/Location.php

```diff
@@ -143,6 +143,18 @@ public function getCategoryList()
         return '';
     }
 
+    /**
+     * @return mixed
+     */
+    public function getWebsite()
+    {
+        $website = $this->getField('Website');
+        if ($website && strpos($website, '//') === false && strpos($website, '.') >= 0) {
+            return '//' . $website;
```

Comment:
Does this work without `http` or `https` spec'd? I think we are OK forcing `http` if there isn't a protocol spec'd.

Reply:
Unless we're prepending `http` somewhere in the specific implementation this came up in.